Cargo.toml

```diff
@@ -3,6 +3,7 @@ default-members = [
   "crates/checkpoint-downloader",
   "crates/walrus-core",
   "crates/walrus-e2e-tests",
+  "crates/walrus-orchestrator",
```

Comment:
Do we need this to be the default member? By removing this, simtests should work again.

Reply:
I don't think we do indeed. Happy to remove it. Why does it make fail the simtests though (out of curiosity)?

Reply:
I replied some details here: https://mysten-labs.slack.com/archives/C079UUZQSM6/p1743487502346859?thread_ts=1742920188.842889&cid=C079UUZQSM6

TLDR is that the aws crate used in the `walrus-orchestrator` added a feature that requires explicit crypto provider to be set.

Reply:
mmh it seems it didn't work

Reply:
The current code doesn't seem to compile. The error in CI is a different error. Or are you still seeing the same `no process-level CryptoProvider available` locally?

Reply:
very strange, checking in a bit

Reply:
It is indeed complaining about something deep inside sui-core; it is a mystery how such a simple PR can trigger so many problems